Support/Scripts/prepare-android-ndk.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+##
+## Copyright (c) 2014-present, Facebook, Inc.
+## All rights reserved.
+##
+## This source code is licensed under the University of Illinois/NCSA Open
+## Source License found in the LICENSE file in the root directory of this
+## source tree. An additional grant of patent rights can be found in the
+## PATENTS file in the same directory.
+##
+
+import sys
+import argparse
+from typing import Any
+import urllib.request
+import zipfile
+import os
+import shutil
+
+try:
+    import progress.bar
+except ImportError:
+    print("python3 -m pip install progress to see progress bars")
```

Comment:
I think people usually do `pip3` instead of `python3 -m pip`.

Reply:
Meh, with chef/brew/Apple deciding to overwrite any of `{python,pip}{,2,3}` whenever they feel like it I'm no longer confident that the first `pip` on my path corresponds to the first `python` on my path.